headlamp-k8s/headlamp
# frontend: clusterActionSlice: Refactor clusterActions into a slice

- Moved clusterAction related stuff from reducers/actions/sagas into clusterActionSlice.
- Added some documentation.
- Added some tests.

## How to test?

- Create or edit some yaml, and it should apply.
- Try cancelling an action with the snackbar cancel buttons.

Part of https://github.com/headlamp-k8s/headlamp/issues/1159 epic.
